content/docs/command-reference/push.md

```diff
@@ -164,7 +164,6 @@ Posts.tsv.dvc
 Posts-test.tsv.dvc
 matrix-train.p.dvc
 model.p.dvc
-Dvcfile
```

Comment:
@jorgeorpinel can we use example-get-started repo here in example.
And also update it with `dvc dag`.